### ISSUE TITLE
Accessibility concerns 

### ISSUE THREAD

**Issue by frob** [edx-solutions/xblock-image-explorer#54]
Has there been an audit of the image explorer with regard to accessibility? 

What are the guidelines for writing accessible content with the image explorer.

**Comment by nasirhjafri** [comment]
@frob Yes we'd an audit and pushed some fixes regarding accessibility. It's still a work in progress.
